src/Sarif/Autogenerated/Fix.cs

```diff
@@ -34,7 +34,7 @@ public SarifNodeKind SarifNodeKind
         /// <summary>
         /// A string that describes the proposed fix, enabling viewers to present a proposed change to an end user.
         /// </summary>
-        [DataMember(Name = "description", IsRequired = true)]
+        [DataMember(Name = "description", IsRequired = false, EmitDefaultValue = false)]
```

Comment:
The change to this autogenerated file is the result of changing the schema to not require the `fix.description` property.